src/views/ConfigEditor.tsx

```diff
@@ -91,7 +99,12 @@ const ConfigEditor = (props: ConfigEditorProps) => {
       <Collapse collapsible label="Access Token Permissions" isOpen={isOpen} onToggle={() => setIsOpen((x) => !x)}>
         <p>
           To create a new Access Token, navigate to{' '}
-          <a className={styles.externalLink} href="https://github.com/settings/tokens" target="_blank" rel="noreferrer">
+          <a
+            className={styles.externalLink}
+            href="https://github.com/settings/tokens?type=beta"
```

Comment:
If we are changing the default url to create access token to be the fine-grained token, we should probably update next steps bellow, as it is different with the new token.  

>  <p>Ensure that your token has the following permissions:</p>
        <b>For all repositories:</b>
        <pre>public_repo, repo:status, repo_deployment, read:packages, read:user, user:email</pre>
        <b>For GitHub projects:</b>
        <pre>read:org, read:project</pre>
        <b>An extra setting is required for private repositories:</b>
        <pre>repo (Full control of private repositories)</pre>